test/appium/views/base_view.py

```diff
@@ -287,6 +299,7 @@ def __init__(self, driver):
         self.connection_status = ConnectionStatusText(self.driver)
         self.cross_icon = CrossIcon(self.driver)
         self.show_roots_button = ShowRoots(self.driver)
+        self.open_in_status_from_external_browser_button = OpenInStatusFromExternalBrowserButton(self.driver)
```

Comment:
too long name, please rename the button without `where?` part, instead of adding this to the element name you can create a specific section marked with comments e.g.
```
# external browser
self.open_in_status = OpenInStatus(self.driver)
```

Reply:
Fixed.